build/android/gyp/create_flutter_jar.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""Create a JAR incorporating all the components required to build a Flutter application"""
```

Comment:
I'd probably put this in `//sky/build` rather than this directory.  Most of the stuff in this directory is cloned from Chromium.  It shouldn't know about flutter.